changelogs/unreleased/1699-mittachaitu

```diff
@@ -0,0 +1 @@
+fix(webhook): reject PVC deletion request when volume has dependent snapshots
```

Comment:
Is this for CSI or regular volumes? Wasn't this already present?

Reply:
> Is this for CSI or regular volumes?

This fix is for existing volumes.

> Wasn't this already present?

We added checks only for clones but not for snapshots.